### ISSUE TITLE
Guide / What am I doing wrong

### ISSUE THREAD

**Issue by Sekuraz** [SmithsGaming/Armory#52]
Hey, 
I am currently playing the Minecolonies Testpack (without AppleSkin) and I wanted to use this Mod, but i have no guide, no wiki and no documentation. Could you please take 5 Minutes to write it down in your posts on curse and on the minecraft forum?

Especially why this is not working:
![Image](https://user-images.githubusercontent.com/7331709/29949358-758dd082-8eb4-11e7-9527-ddc538fde13a.PNG)

regards,
Sekuraz

**Comment by Sekuraz** [comment]
@OrionDevelopment

**Comment by marchermans** [comment]
I have noticed, i am actually debugging this. This should be working...... Does NEI work for you?

**Comment by Sekuraz** [comment]
Yes, I took the recipes from there, This is why i disabled AppleSkin.

**Comment by marchermans** [comment]
This should work.........
Let me run some tests tomorrow. This is 1.10 you say?

**Comment by marchermans** [comment]
Does it work for any other request?

**Comment by Sekuraz** [comment]
Not as far as I can tell. It is 1.10, according to the Testpack, which should be 1.10
